src/foxops/engine/rendering.py

```diff
@@ -14,6 +15,20 @@
 logger = get_logger(__name__)
 
 
+def first_ip_address_is_greater(first_ip, second_ip):
```

Comment:
what do we need this for?

Also - if we need it, I would probably adapt name to make the usage look a bit more natural:

```
{{ ip | ip_address_is_greater_than(second_ip) }}
```

Reply:
Name adapted. We want to be sure that we are providing a valid range of IPs. Otherwise we could have an invalid IPAddressPool.
possible error situations for our use case:
- range is provided in an incorrect order: 1.1.1.4-1.1.1.1
- range does not have at least 3 IPs: 1.1.1.1-1.1.1.2

Reply:
As we agreed, I have removed this function for now.